SmartDeviceLinkTests/DevAPISpecs/SDLStreamingVideoLifecycleManagerSpec.m

```diff
@@ -32,15 +33,21 @@
 #import "SDLVideoStreamingState.h"
 #import "TestConnectionManager.h"
 
+@interface SDLStreamingVideoLifecycleManager ()
+@property (copy, nonatomic) NSString *appName;
+- (NSString *)videoStreamBackgroundString;
```

Comment:
This needs to update along with the change to this method (by turning it into a property) above

Reply:
Fixed